omod/src/main/webapp/pages/visit/templates/encounters/hivNotMigratedEncounterLong.gsp

```diff
@@ -0,0 +1,30 @@
+<div class="header" ng-include="'templates/encounters/defaultEncounterHeader.page'">
+</div>
+
+<div class="content encounter-summary-long">  <!-- encounter-summary-long currently only used for Selenium tests -->
+    <p class="bold">
+        ${ ui.message("pihcore.hiv.legacyDataNotMigrated") }
+    </p>
+    <p ng-repeat="obs in encounter.obs" class="aligned">
+        <label>{{ obs.concept | omrsDisplay }}</label>
+        <span class="value">{{ obs | obs:"value" }}</span>
+    </p>
+    <div ng-show="encounter.orders && encounter.orders.length">
+        <span ng-repeat="order in encounter.orders">
+            <p class="aligned">
+                <label>${ ui.message("coreapps.patientDashBoard.orderNumber")}</label>
+                <span class="value">{{ order.orderNumber }}</span>
```

Comment:
As far as our migrated data is concerned, no order numbers are meaningful.  For drug orders (which are the only orders we are migrating into afaik), this is a generated number that we are inventing just so this is non-null.  So I'd prefer we not include it.  I also don't think urgency is particularly useful (below).  Just displaying "effectiveStartDate" will give you either the date already started or the date scheduled to start (whether that is in the past or future) which is typically what we want to see I think.   I don't know what "order.display" is - I'm not aware of any "display" method or property on Order.  I'd think here we'd want to show the orderable (concept).